src/features/home/testimonials/Testimonials.tsx

```diff
@@ -0,0 +1,149 @@
+    subject: 'Secondary (11 to 14 years) Chemistry tuition',
+  },
+  {
+    name: 'Jane Ross',
+    feedback: `Asma is an excellent tutor and I highly recommend her to anyone struggling in Physics, Chemistry and Maths. 
+      She helped my daughter gain a newfound enjoyment and confidence in the subjects that Asma tutored her on.`,
+    subject: 'GCSE or National 3-5 Chemistry tuition',
+  },
+  {
+    name: 'Rizwan Ahmad',
+    feedback: `Asma has been brilliant for my daughter. When we started tuition for physics, my daughter had lots of 
+      difficulties and at the end, she scored 9 in GCSE, which tells all about Asma. I would definitely recommend Asma. 
+      Thanks for your hard work.`,
+    subject: 'GCSE or National 3-5 Physics tuition',
+  },
+  {
+    name: 'Lawunmi Adekunte',
+    feedback: `If you are looking for a tutor who will guide, teach, and challenge you, look no further! Asma is an 
+      excellent tutor because she is patient and tailored the learning to my needs. I joined Asma when I was not confident in 
+      maths and not getting good marks. Now, I have achieved a grade 9 in my GCSE maths and I will be taking A level maths. 
+      I’m so grateful for all the hard work and effort Asma put into each of our lessons. Thank you so much Asma!`,
+    subject: 'Secondary (11 to 14 years) Maths tuition',
+  },
+  {
+    name: 'Maureen Macmillan',
+    feedback: 'Asma is an excellent tutor. She makes maths and physics easy to understand.',
+    subject: 'Secondary (11 to 14 years) Physics tuition',
+  },
+  {
+    name: 'Elizabeth Stone',
+    feedback: `I am extremely happy, grateful and honoured that my child was taught by Asma. Despite Asma not having a 
+      dyslexic teaching background, my child was able to develop confidence, interest, and high grades in all three sciences 
+      thanks to Asma's tailored and detailed teaching. I highly recommend her.`,
+    subject: 'GCSE or National 3-5 Chemistry tuition',
+  },
+  {
+    name: 'Jing Han',
+    feedback: `Very good one, my boy likes the lessons. Asma worked with our son in preparation for his GCSE Maths exam. 
+      Asma provided excellent weekly support, our son enjoyed the sessions and completed additional homework set by Asma. 
+      We were all delighted by his recent mock exam results which he reached with Asma's excellent tutoring.`,
+    subject: 'Secondary (11 to 14 years) Chemistry tuition',
+  },
+  {
+    name: 'Eddy Yang',
+    feedback: `We are delighted that we found Asma. She is very organised and kind, and we have seen marked progress for our 
+      son. His confidence in maths has increased along with his marks. He scored significantly higher in his GCSE mocks 
+      after just six months of tuition with Asma.`,
+    subject: 'A-Level, IB or Highers Maths tuition',
+  }
+];
+
+const Testimonials = () => {
+  return (
+    <TestimonialsWrapper className="testimonials-swiper">
```

Comment:
Again, as stated above, all of that css can be incorporated inside this styled component